lib/index.js

```diff
@@ -307,7 +307,7 @@ function saveRecord (params, callback) {
   });
   var countdown = filtered.length;
   if (countdown === 0) {
-    return callback();
+    return callback(new Error('no valid items to save!'));
```

Comment:
I don't think this is necessarily an error case. If there's nothing to save then it's fine, just move on.